defender/tests.py

```diff
@@ -532,6 +532,16 @@ def test_admin(self):
         from .admin import AccessAttemptAdmin
         AccessAttemptAdmin
 
+    def test_unblock_view_user_with_plus(self):
+        """
+        There is an available admin view for unblocking a user
+        with a plus sign in the username.
+
+        Regression test for #GH76.
+        """
+        reverse('defender_unblock_username_view',
```

Comment:
This just checks it doesn't fail, because if there is no view available a `NoReverseMatch` is raised.